src/instruments/src/RMP/index.jsx

```diff
@@ -0,0 +1,30 @@
+import ReactDOM from 'react-dom';
+import { renderTarget } from '../util.mjs';
+import './style.scss';
+import { useState } from 'react/cjs/react.production.min.js';
+import { RadioManagementPanelPair } from './RadioManagementPanel.mjs';
+
+
+const radioManagementPanelPair = new RadioManagementPanelPair();
```

Comment:
I'm generally not stoken about globals like this that are used by React code, but kept outside of React. They can make testing hard in future when we have it and where usually you can just introduce props around or clearly observe React state, these live outside of the tree making both those use cases harder than they need to be.

An alternative pattern could be to keep the class instance in React state itself:

```
const [rmp, _] = useState(() => new RadioManagementPanelPair());
```